addon/components/layout/left-menu/component.js

```diff
@@ -69,33 +69,32 @@ export default Component.extend({
   }),
 
   streamsURL: computed('analyticsURL', function () {
-    let module = getOwner(this).resolveRegistration('config:environment').modulePrefix;
+    let module = getOwner(this).resolveRegistration('config:environment').APP.name;
```

Comment:
@Hegazy360 i think you should consider using the router service to know in which app you are, it will allow you to avoid going into all of the apps to change the APP.name

Reply:
I noticed that APP.name was already set correctly that's why I chose to use it, for acquisition at least it was @upfluence/acquisition-web